vagrantfile-windows.template

```diff
@@ -4,6 +4,7 @@
 Vagrant.configure(2) do |config|
   config.vm.guest = :windows
   config.vm.communicator = "winrm"
+  config.vm.network :forwarded_port, guest: 3389, host: 3389, id: 'rdp', auto_correct: true
```

Comment:
the host should probably be something like 33389 in case you are on a windows host.